cli/exp_scaletest.go

```diff
@@ -1142,27 +1168,27 @@ func (r *RootCmd) scaletestDashboard() *clibase.Cmd {
 	}
 
 	cmd.Options = []clibase.Option{
-		{
-			Flag:        "count",
-			Env:         "CODER_SCALETEST_DASHBOARD_COUNT",
-			Default:     "1",
-			Description: "Number of concurrent workers.",
-			Value:       clibase.Int64Of(&count),
-		},
 		{
 			Flag:        "min-wait",
 			Env:         "CODER_SCALETEST_DASHBOARD_MIN_WAIT",
-			Default:     "100ms",
+			Default:     "1s",
 			Description: "Minimum wait between fetches.",
 			Value:       clibase.DurationOf(&minWait),
 		},
 		{
 			Flag:        "max-wait",
 			Env:         "CODER_SCALETEST_DASHBOARD_MAX_WAIT",
-			Default:     "1s",
+			Default:     "10s",
```

Comment:
Any chance we may be able to eliminate these options? It'd be nice if you didn't need to adjust these parameters to have a working load test, and that it'd continue working even if coderd was under too much load, etc.

We could instead track user experience impact when things slow down to a crawl.

Reply:
This more just controls the interval between actions. Between 1 and 10 seconds I think is a reasonable default. I'll rename it to better reflect its semantics.